extensions/data-transfer/portability-data-transfer-generic/src/test/java/org/datatransferproject/datatransfer/generic/GenericImportSerializerTestBase.java

```diff
@@ -0,0 +1,33 @@
+package org.datatransferproject.datatransfer.generic;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+import org.junit.BeforeClass;
+
+class GenericImportSerializerTestBase {
+  static ObjectMapper objectMapper = new ObjectMapper();
+
+  @BeforeClass
+  public static void onlyOnce() {
+    GenericImporter.configureObjectMapper(objectMapper);
+  }
+
+  <T> void assertJsonEquals(String expectedPayload, GenericPayload<T> actualWrapperPayload)
```

Comment:
This seems really useful but the longer instances where this is used were still somewhat ugly and hard to write with all the quotes, quote escapes, and line concatenations.  Is it easy for assertJsonEquals to handle that too and make these tests even easier to write/read?  And is this another utility that is potentially used elsewhere in DTP?

Reply:
Agreed, this is another one where the python equivalent is much simpler!
Mutli-line string literals weren't added until Java 15 (13 with a compiler flag), and we're still on 11 so annoyingly can't utilise those and I think this is the best we can do for embedding large JSON strings in java files.
Looking at other tests, the imgur importer tests store [expected JSON data in resource files](https://github.com/dtinit/data-transfer-project/blob/6705a21f3ae691cc5ff6abd9c781717448074da4/extensions/data-transfer/portability-data-transfer-imgur/src/test/resources/all_images.json#L4) which would be another option, but it does mean the data doesn't live in the test file itself

I could be convinced either way on this, and I'd be open to other suggestions if anyone has them